pkg/create/templates/test.go

```diff
@@ -106,7 +114,9 @@ test_connection() {
   local result=1
   while [ $attempt -le $max_attempts ]; do
     echo "Sending GET request to http://$(container_ip):${test_port}/"
-    response_code=$(curl -s -w %{http_code} -o /dev/null http://$(container_ip):${test_port}/)
+    # here, we run the curl inside a container for people running on boot2docker (who can't access the container's IP directly)
+    #response_code=$(curl -s -w %{http_code} -o /dev/null http://$(container_ip):${test_port}/)
+    response_code=$(docker run --rm -it sequenceiq/alpine-curl curl -s -w %{http_code} -o /dev/null http://$(container_ip):${test_port}/)
```

Comment:
I'm a bit worried this introduces yet another dependency `sequenceiq/alpine-curl` which is not desired in tests, imho.